raml/raml-information.js

```diff
@@ -73,7 +73,60 @@ var RamlInformation = null;
             return false;
 
         }
-    }
+    };
+
+    /**
+     * This method returns the contentType of the request for the endpoint
+     * @param endpointUri
+     * @returns {null}
+     */
+    RamlInformation.prototype.getContentTypeForPostEndpoint = function (endpointUri) {
+
+        var typeObject;
```

Comment:
Kann man das dich nicht in eine Methode packen? die beiden Methoden machen doch genau das selbe nur für andere request "methods"